Backpack/Calendar/Classes/BPKCalendarHeaderCell.m

```diff
@@ -0,0 +1,33 @@
+//
+//  BPKCalendarHeaderCellCollectionViewCell.m
+//  Backpack
+//
+//  Created by Aron Szabados on 2018. 11. 16..
+//
+
+#import "BPKCalendarHeaderCell.h"
+#import "BPKCalendarAppearance.h"
+
+#import <Backpack/Color.h>
+#import <Backpack/Font.h>
+#import <Backpack/Spacing.h>
+
+@implementation BPKCalendarHeaderCell
+
+- (instancetype)initWithFrame:(CGRect)frame {
+    self = [super initWithFrame:frame];
+    if (self)
+    {
+        [self.titleLabel setTextAlignment:NSTextAlignmentLeft];
+    }
+    return self;
+}
+
+- (void)layoutSubviews {
+    [super layoutSubviews];
+    BPKFontStyle fontStyle = ((BPKCalendarAppearance *)self.header.calendar.appearance).headerTitleFontStyle;
```

Comment:
This needs an assert to ensure we don't break it accidentally. i.e. assert that the type is correct

Reply:
Assert added